real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/logging/OldLogConverter.java

```diff
@@ -0,0 +1,183 @@
+package com.roboclub.robobuggy.logging;
+import java.util.*;
+import java.io.*;
+
+
+public class OldLogConverter {
+	
+/** Steering Angle Conversion Rate */
+static private final int ARD_TO_DEG = 100;
+/** Steering Angle offset?? */
+static private final int OFFSET = -200;
+	
+static private ArrayList<String> sensor_type = new ArrayList<String>();
+static private ArrayList<Integer> sensor_quant = new ArrayList<Integer>();
+
+	public static void main(String[] args) throws IOException {
+		boolean fileinput = false;
+		Scanner scanner; PrintStream writer;
+		
+		if(fileinput){
+			scanner = new Scanner(System.in);
+			System.out.println("Input filepath to old data file.");
+			String inputpath = scanner.nextLine();
+			System.out.println("Input filepath to new data file.");
+			String outputpath = scanner.nextLine();
+			scanner.close();
+			scanner = new Scanner(new File(inputpath));
+			writer = new PrintStream(new File(outputpath));
+		}
+		else{
+			scanner = new Scanner(new File("/Users/davidneiman/Desktop/sensors.txt"));
```

Comment:
Can we try to make these relative file paths? Maybe put it in a folder called "convertedLogs"?